_posts/2018-09-10-remotiquette-at-store2be.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2018-09-10
+categories:
+  - Remote
+  - Workflow
+  - Work Ethics
+title: Remotiquette at store2be
+author_staff_member: 06_sooraj
+image: /images/store2be-digital-nomads.jpg
+---
+
+store2be provides you with the best locations for your roadshows or promotions.
+
+Our mission:
+
+> Build a global booking platform that enables brands to book scalable, measurable and targeted Live Marketing.
+
+Our tech team is often spread across the globe and this post gives a sneak peak to how our day looks like, store2be remote culture, and how we manage to work across time zones.
+![Digital Nomad- image courtesy: Pexels.]({{ site.url }}/images/store2be-digital-nomads.jpg)
+
+## Setting the culture
+
+Culture is more than free food, coffee, and foosball tables. Our definition of fun at work doesn’t necessarily mean playing ping pong all day,rather have fun writing code, you enjoy discussing new frameworks, you enjoy tweaking that UI a bit.
+
+## Communication
+
+In remote teams culture === communication. Communication is the most important aspect of any remote team, and the way you communicate with each other dictates your culture.
+
+At store2be, we use Slack for communication. We try to stay in touch with the team and show our presence.
+
+Be nice, greet the team when you start (irrespective of the timezone), react to messages with emojis even if the message might not be super relevant to you. Let the team know when you are out for lunch or a break.
```

Comment:
What do you think about adding something like "when going through slack" to the "react to messages with emojis even if the message might not be super relevant to you" part? So that it doesn't sound like we expect people to go through slack adding emojis, I understand what you mean 100% but in writing for others I personally think it's better to specify a bit more

Reply:
makes a lot of sense.